### PR TITLE
feat: [M1679] Implement download data button on dashboard

### PR DESCRIPTION
**Pull request by tienhoah**

### CODE REVIEW

**Comment by tienhoah** [comment]
[implement download data button on dashboard](https://trello.com/c/dvS94HGI/1679-implement-download-data-button-on-dashboard)

File: src/constants/transect-protocols.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const protocolsArray = [
+export const availableProtocolsForChart = [
```
Comment (mmnoo) [inline src/constants/transect-protocols.js:1]: nice rename :)

File: src/components/TooltipLayout.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Tooltip from '@material-ui/core/Tooltip';
+import Fade from '@material-ui/core/Fade';
+
+const TooltipLayout = ({ title, placement, children }) => {
+  return (
+    <Tooltip
```
Comment (mmnoo) [inline src/components/TooltipLayout.js:7]: Oh ya, MUI! Saves so much time!

File: src/components/SiteDetailSubItems.js [hunk 3]
```diff
@@ -1,19 +1,33 @@
...
 import { theme } from '../constants/theme';
+import { withStyles } from '@material-ui/core/styles';
 import ContactIcon from '@material-ui/icons/Email';
 
+import Button from '@material-ui/core/Button';
 import Typography from '@material-ui/core/Typography';
 import Box from '@material-ui/core/Box';
+import DownloadDataModal from './DownloadDataModal';
+import { useMediaQuery } from '@material-ui/core';
+import TooltipLayout from './TooltipLayout';
```
Comment (mmnoo) [inline src/components/SiteDetailSubItems.js:13]: Im not sure about the name `TooltipLayout` its not really a layout component, just a custom tooltip. Could it be called `MermaidTooltip` or `MermaidTooltip1` if there are others, or `MermaidDashboardTooltip` if its different than the collecting app?.

File: src/components/DownloadDataModal.js [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+const MermaidButton = withStyles({
+  root: {
+    margin: '4px',
+    color: theme.cardButton.color.mermaidWhite,
+    backgroundColor: theme.cardButton.bgColor,
+    '&:hover': {
+      backgroundColor: theme.cardButton.bgColor
+    }
+  }
+})(Button);
```
Comment (mmnoo) [inline src/components/DownloadDataModal.js:43]: Is this a one-off styling of this button? If not, is it worth it to make it a reusable component?

File: src/components/DownloadDataModal.js [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+const TableHeadWrapper = styled(TableHead)`
+  background-color: lightgrey;
+`;
+
+const TableCellWrapper = styled(TableCell)`
+  text-transform: capitalize;
+`;
+
+const DownloadDataModal = ({ currentSelectedSite }) => {
+  const mediaMin1281 = useMediaQuery('(min-width:1281px)');
```
Comment (mmnoo) [inline src/components/DownloadDataModal.js:54]: Hm, this feels very css-y, and maybe not appropriate for JS/markup, but taht depends on the tradeoffs of the project.

**Review by mmnoo — APPROVED** [review]

**Review by mmnoo — COMMENTED** [review]
Nice. Approved if you remove the console.logs you forgot :)

**Pull request merged**